scripts/compsets/config/wcoss2.config

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+### WCOSS2 SETUP ###
+
+## some lsf/user defaults, should be loaded already but need some defaults just in case
+export ACCOUNT=${ACCOUNT:-"WFS-DEV"}
+export QUEUE=${QUEUE:-"dev"}
+
+## initialize modules
+module reset
+module use -a $BASEDIR/modulefiles/$BUILD_TARGET
+module load wam-ipe
+module list
+## computational stuff
+export TPN=${TPN:-128}
+export OMP_NUM_THREADS=${OMP_NUM_THREADS:-1}
+
+## system directories
+export STMP=${STMP:-/lfs/h1/swpc/stmp}
+export PTMP=${PTMP:-/lfs/h1/swpc/ptmp}
+
+## executables/scripts
+export SIGHDR=${SIGHDR:-/lfs/h1/swpc/wam/noscrub/adam.kubaryk/wam-ipe_workflow/sorc/wamipe_utils.fd/exec/global_sighdr}
+export SFCHDR=${SFCHDR:-/lfs/h1/swpc/wam/noscrub/adam.kubaryk/wam-ipe_workflow/sorc/wamipe_utils.fd/exec/global_sfchdr}
+export NEMSIOGET=${NEMSIOGET:-/lfs/h1/swpc/wam/noscrub/adam.kubaryk/wam-ipe_workflow/sorc/wamipe_utils.fd/exec/nemsio_get}
```

Comment:
same here

Reply:
this specifically cannot be used in ops, not worried about this one.